pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -39,6 +55,28 @@ def test_arrow_roundtrip(data):
     tm.assert_frame_equal(result, df)
 
 
+@td.skip_if_no("pyarrow")
+def test_dataframe_from_arrow_type_mapper(int_dtype):
```

Comment:
can you use this fixture: any_int_ea_dtype and remove the int_dtype from above

Reply:
Those are strings, but `types_mapper` expects actual dtype objects. Maybe there's a way to convert, though?

Reply:
`pandas.api.types.pandas_dtype` is the function I needed.